src/i18n/index.ts

```
@@ -74,6 +77,11 @@ export async function getComponentI18nResources(
     } catch (_) {
       resources = null;
     }
+    if (!resources) {
+      console.error(
+        `No localized string available for "${componentName}" for "${locale}" locale. Falling back to English translation.`
+      );
```

Thinking about this more, I'm not sure we should log errors for this (maybe not for the other things in this file as well).  It could lead to noise in app developers' logs that lead to a distraction when debugging and diagnosing their own bugs.

I'm also a little worried that failing on any console error could bite us down the line. What if we want to make sure an error is logged in a certain scenario? 

Can we do something more targeted here, like creating isolating the report of the missing translation key to a function which we mock when testing to throw an exception instead of logging?